packages/esm-patient-chart-app/src/side-nav/side-menu.component.tsx

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import styles from './side-menu.scss';
 import { SideNav, SideNavProps } from 'carbon-components-react';
-import { ExtensionSlot, useLayoutType } from '@openmrs/esm-framework';
-import { useNavMenu } from './useNavMenu';
+import { useLayoutType } from '@openmrs/esm-framework';
 import { isDesktop } from '../utils';
+import PatientChartNavMenu from './nav.component';
 
 interface SideMenuPanelProps extends SideNavProps {}
 
 const SideMenuPanel: React.FC<SideMenuPanelProps> = () => {
   const layout = useLayoutType();
-  useNavMenu();
 
   return (
     isDesktop(layout) && (
       <SideNav expanded aria-label="Menu" className={styles.link}>
-        <ExtensionSlot extensionSlotName="nav-menu-slot" />
+        <PatientChartNavMenu />
```

Comment:
Simplified this. The idea was to make the left nav change contents based on what page you're on. However at present the left nav only exists on the patient chart. Once we have left nav on other pages, we can re-introduce the abstraction, but using unistore rather than the extension system.